website/addons/wiki/static/pagedown-ace/Markdown.Editor.js

```diff
@@ -1704,9 +1721,13 @@ var Range = ace.require('ace/range').Range;
 
             buttons.redo = makeButton("wmd-redo-button", getStringAndKey("redo"), "-220px", null);
             buttons.redo.execute = function (manager) { inputBox.session.getUndoManager().redo(); };
+
             makeSpacer(4);
             makeCheckBox("wmd-autocom-toggle", "autocom", "-240px", "Autocomplete");
 
+            makeSpacer(5);
+            buttons.help = makeHelpButton("wmd-help-button", getStringAndKey("help"), "-240px");
+
             if (helpOptions) {

```

Comment:
I'm not sure who made the conditional or why it is that way, but originally the button was supposed to appear when the option for the conditional was true. 

Someone somewhere never passed in the conditions to make it true and I was told to make a modal pop-up for the button so I just made a new function that made the modal and created the button. I didn't want to delete someone else's code when I wasn't sure where else in the code it might be called so I just left it there even if it is never made to be true.